### PR TITLE
Fix #76 : Add blurb to config.json

### LINKED ISSUE #76: Add blurb to config.json
Issue https://github.com/exercism/java/issues/759 is also related this track:
> In config.json we need to have a blurb which explains a bit about Java in ~30 words. See the Go track's config.json for an example.

### PR DESCRIPTION
**Pull request by Dispader**
I created the blurb here by starting with the first-sentence introduction on the Apache Groovy site, and attempting to winnow it down to 30 characters.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "language": "Groovy",
   "checklist_issue": 9,
   "active": false,
+  "blurb": "Apache Groovy is an optionally typed, optionally dynamic, object-oriented and functional language which aims to improve developer productivity and happiness.  It is fully compatible with the JVM ecosystem.",
```
Comment (kytrinyx) [inline config.json]: If possible, let's see if we can get something more conversational here. The "optionally typed / optionally dynamic" and "object-oriented and functional" make this pretty difficult to grasp.

Here are three blurbs that I find both conversational and intriguing:

The R blurb reads:

> R is an interpreted, open source programming language with powerful data manipulation and graphical capabilities. It has a thriving package ecosystem and is one of the most popular languages for data science.

The Ruby blurb reads:

> Ruby is a dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write.

The Go one reads:

> Go is a compiled, open source programming language with a small, consistent syntax, a powerful standard library, and fantastic tooling. It's a great fit for web backends and command-line tools.

Some questions that might help find the tone and contents for the Groovy blurb.

* What is Groovy typically used for?
* What about the language makes people happy? (In Ruby it's _a syntax that is natural to read and easy to write_, whereas for Go it's _small, consistent syntax_).
* What pain points was Groovy specifically trying to address when the creators started working on it?
* What do you like about writing Groovy?
* If you were out having a drink with a programmer friend who had never written Groovy, what would you tell them to get them interested in the language?

Feel free to riff on these in verbose ways, we can winnow down once we find a direction.

Comment (Dispader) [reply config.json]: @kytrinyx Thank you so much!  I'm wowed by how awesome and insightful that post is— and thank you for taking the time.

I'm going to reflect for a while before blurting out my thoughts on the great questions above.  (I'd love to hear @m-a-ge 's thoughts, as well.)

Comment (m-a-ge) [reply config.json]: @kytrinyx, thanks for your explanation. The description is a bit formal, we could come with something else.

For me **Groovy** is an unopinionated multi-paradigm language which enhances Java and makes you code less verbose. It made me happy working with `xml` (could you imagine that?) and I love its `groovyConsole` (similar to Python`s `IDLE`) which allows you to try out some code snippet very fast (in comparison with Java, but I guess Java 9 somewhat solve this). You can also write Groovy scripts (like in Python) which is awesome!
Groovy is an open source language which is totally dependent on JVM, we should mentioned that as well.

@Dispader, let's come up with our own description, what do you think? 

P.S. Sorry for the delay, my arm is finally healed 🙃

Comment (kytrinyx) [reply config.json]: > For me Groovy is an unopinionated multi-paradigm language which enhances Java and makes you code less verbose.

This is great.

> It made me happy working with xml (could you imagine that?)

Haha, only barely!

> let's come up with our own description

🎉

Comment (m-a-ge) [inline config.json]: I propose the following text:

"Groovy is an unopinionated multi-paradigm language that enhances Java with intent to make your code less verbose but still utilizing the power of JVM"

'Invented' it myself, so feel free to correct something

Comment (kytrinyx) [reply config.json]: I love it!

Comment (Dispader) [reply config.json]: @m-a-ge @kytrinyx Hear, hear!

**Comment by Dispader** [comment]
I added a very slightly modified version of @m-a-ge 's most excellent blurb, and I'm pretty happy with it.

Right now it reads:

>Groovy is an unopinionated, multi-paradigm language that enhances Java with the intent to make your code less verbose, yet still allows you utilize the power of the JVM.

I'll throw another edit out there, with a little less stress on the Java-compatability:

>Groovy is an unopinionated, multi-paradigm language that lets you use all the power of JVM and it's libraries, with less verbose code than many JVM languages.

But, I think the first reads great, and is a massive improvement!  What do ya fine folks think?

**Comment by kytrinyx** [comment]
I like the first one better. JVM is kind of "jargon-y", and is probably better suited to the ABOUT.md description.

**Comment by m-a-ge** [comment]
@Dispader thanks!
The second variant is more accurate, but the first one is clearer to newcomers because of the following phrase ` that enhances Java`, it was the key idea
I'm voting for the first one for that reason

**Comment by m-a-ge** [comment]
I went ahead and merged the PR
Thanks, @Dispader!

**Pull request merged**